### PR TITLE
order update shipment

### PR DESCRIPTION
**Pull request by scottcrawford03**
fixes feature specs failing as a result of original pr that was reverted:

https://github.com/bonobos/solidus/pull/25

### CODE REVIEW

**Comment by jordan-brough** [comment]
What changed that fixed the feature specs?

**Comment by scottcrawford03** [comment]
@jordan-brough richard has been working on getting the feature specs to work overall. I just made a change in `backend/spec/features/admin/orders/new_order_spec.rb` to fix the feature spec as it relates to my changes. I had to simulate the signifyd approval and readying of shipments

File: backend/spec/features/admin/orders/new_order_spec.rb [hunk 1]
```diff
@@ -52,6 +52,8 @@
     click_on "Payments"
     click_icon "capture"
 
+    Spree::Order.last.shipments.each(&:ready!) #simulate signifyd approval and readying of shipments
```
Comment (scottcrawford03) [inline backend/spec/features/admin/orders/new_order_spec.rb:55]: @jordan-brough this line is what changed

**Comment by jordan-brough** [comment]
Nice. :+1:

**Pull request closed without merging**